scripts/services/redis-tls/generate-cert.sh

```diff
@@ -0,0 +1,66 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+#!/bin/bash
+
+directory=$(pwd)/scripts/services/redis-tls/ssl
+
+mkdir -p $directory
+
+          # Create CA
+
+openssl req \
```

Comment:
I don't like the idea to generate crt everytime. Our developer may don't have the enviorment to do this.

Reply:
But you can't guarantee the container ip always in 172.30.0.0/24 like https://github.com/apache/incubator-opendal/pull/2858/files#diff-66d31d862c59877b8e265e0b23baea97975bbcfa336076b0eeaf24711d9b9c5dR5-R13